integration-tests/package.json

```diff
@@ -4,13 +4,7 @@
   "description": "A wrapper module that can install and run Realm JS tests in various environments",
   "private": true,
   "scripts": {
-    "start": "npm start --prefix tests --",
-    "test": "npm run build --prefix tests && concurrently -m 1 npm:test:*",
-    "test:node": "npm test --prefix environments/node",
```

Comment:
The readme should be updated to reflect these changes

Reply:
Ahh. Thanks I got away from that.
... I wonder why I deleted those 🤔

Reply:
I added them back in.. I started this branch trying to do something else ...